### ISSUE TITLE
Testing: When typing to input field, validation is not triggered

### ISSUE THREAD

**Issue by GrazingScientist** [zauberzeug/nicegui#3759]
### Description

I am currently evaluating, that my input field validators are raising as expected.

Here is a simple example:

```py
from nicegui import ui


def setup() -> None:
    @ui.page("/")
    def main() -> None:
        validator = {"Not a number": lambda value: value.isnumeric()}
        ui.input("Number", validation=validator)


# works totally fine in browser
#setup()
#ui.run(port=8888)

# Test that looks for the validation error message fails
async def test_validation(user):
    setup()

    await user.open("/")

    # I provide a non-numeric value, expecting the validator to trigger
    user.find("Number").type("foo")

    await user.should_see("Not a number")
```

When run this in pytest, I get the following error:
> FAILED tests/ui/foo.py::test_validation - AssertionError: expected to see at least one element with marker=Not a number or content=Not a number on the page

But when calling this in the browser, the error message that this is not a valid number comes up. So, that means for me that the `type` function does not appropriately trigger the validation process.

**Comment by rodja** [comment]
Thanks for bringing this up @GrazingScientist. I fixed it in #3765 which will hopefully be released in the next days.

**Comment by GrazingScientist** [comment]
Amazing! Thank you so much, @rodja ! 🤩
